CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Apache Airflow Provider for Great Expectations
 
+## 0.0.5
+* [BREAKING] Updated GreatExpectations operator to work with class-based Checkpoints (Great Expectations >= 0.13.8)  
```

Comment:
I guess that should be 0.13.9 since we're pinning that... even though the checkpoints were released in 0.13.8.
I'd probably add a note to the readme explaining this.